Gemfile

```diff
@@ -19,8 +22,15 @@ gem 'webpacker', '~> 5.0'
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.7'
-
 gem 'rexml', '~> 3.2.5'
+# use to define roles
+gem 'rolify', '~> 6.0'
+# define permissions
+gem 'cancancan', '~> 3.3'
+# fixes indentation
+gem 'rubocop', '~> 1.24', require: false
+# debugger gem
+gem 'pry', '~> 0.14.1'
```

Comment:
Please move this gem to the `development` and `test` group as we don't need to install this gem in the production environment.

Reply:
ok